code/modules/projectiles/projectile.dm

```diff
@@ -1296,7 +1299,11 @@ So if we are on the 32th absolute pixel coordinate we are on tile 1, but if we a
 	if(!P.ammo.ping)
 		return
 	if(prob(65))
-		if(P.ammo.sound_bounce) playsound(src, P.ammo.sound_bounce, 50, 1)
+		var/pitch = 0
+		if(P.ammo.flags_ammo_behavior & AMMO_SOUND_PITCH)
+			pitch = 55000
```

Comment:
This shouldn't run in the ammo doesn't have the relevant sound to begin with. Also you could probs just chuck it in the playsound line itself.